src/controller/util/getTemplateValues.ts

```diff
@@ -9,8 +9,9 @@ import {
 } from "../../config";
 
 import { logger } from "../../util/logger";
+import { configuredCounterMetric, technicalAgeMetric } from "../../util/metrics";
 
-export const getTemplateValues = (req: Request): { [key: string]: any } => {
+export const getTemplateValues = (req: Request, type: string = "3rd-party"): { [key: string]: any } => {
```

Comment:
I do not get the meaning of the `type` parameter? .. as far as I can see this is only used as an additional annotation for metrics. - But cannot we always set this to the same / hence use a constant?

Reply:
This is only to label the metrics whether the traffic comes through a device which supports iframes. 3rd-party(cookie) / iframe